classwrap.d.ts

```diff
@@ -0,0 +1,17 @@
+export as namespace classwrap
+
+export interface ClassSet {
+  [key: string]: boolean | any | ClassSet
+}
+
+export type Class = string | number | ClassSet
+
+/**
+ * Joins all elements of an array or keys of an object into a string.
```

Comment:
Let's keep the description simpler for now. 

> Joins all elements of an array or keys of an object into a string.

I think is good enough. :)

Reply:
This is only a copy/paste of the readme o/

Will change it